teachbooks/external_content/bib.py

```diff
@@ -29,7 +29,7 @@ def read_bibfile(file: Path) -> list[BibEntry]:
     Returns:
         List of .bib file entries.
     """
-    with file.open("r") as f:
+    with file.open("r",encoding='utf-8') as f:
```

Comment:
```suggestion
    with file.open("r", encoding="latin-1", errors="replace") as f:
```
https://docs.python.org/3/library/codecs.html#error-handlers

Dit is misschien de meest nette manier om hiermee om te gaan.

Wanneer de samengevoegde .bib weggeschreven wordt moeten we nog wel specificeren dat dat in latin1 moet.

Reply:
Hm. Blijkbaar doet `errors="replace"` niks bij latin1 omdat alle bytes een character hebben. Hoort dus niet nodig te zijn.